examples/mnist/src/mnist.rs

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
:D

Reply:
haha, this mod would be just downloading the mnist dataset and provide a loader